pypeit/edgetrace.py

```diff
@@ -4220,6 +4228,8 @@ def from_tslits_dict(cls, tslits_dict, master_key, master_dir):
         this.spat_msk = np.zeros(this.spat_cen.shape, dtype=this.bitmask.minimum_dtype())
         this.spat_err = np.zeros(this.spat_cen.shape, dtype=float)
         this.spat_img = np.round(this.spat_cen).astype(int)
+        # JFH This does not work.
+        #this.is_synced = True
```

Comment:
It's not a toggle.  If the instance created by the `from_tslits_dict` isn't synced, there may be a problem in how it's being setup. If so, I can help figure out why.